### PR TITLE
Change base image of PMD from Ubuntu to Debian #1424

### PR DESCRIPTION
**Pull request by Jeeppler**
- change the base image from Ubuntu to Debian #1424
- add README.adoc
- upgrade PMD
- use shared resources
- remove ubuntu references in scripts

closes: #1424

### CODE REVIEW

**Comment by Jeeppler** [comment]
@zigfridus please have a look as well. Currently, I am not able to add you as a reviewer. Maybe, if you comment something I will be able to.

**Comment by zigfridus** [comment]
@Jeeppler  please add me as reviewer.

**Comment by zigfridus** [comment]
@Jeeppler 
I tested these scripts
- 01-start-docker-compose.sh
- 05-start-single-sechub-network-docker-compose.sh
- 50-start-multiple-docker-compose.sh

with this test file
`01-test.sh CWE506_Embedded_Malicious_Code.zip`
They works well.

But there is a problem with 51-start-multiple-object-storage-docker-compose.sh. These are some of errors from scanning:
1. com.mercedesbenz.sechub.commons.model.SecHubRuntimeException: Was not able to upload job data because of IO problems;
2. Caused by: java.io.IOException: Store of sourcecode.zip to s3 failed

I got this result:
[result.zip](https://github.com/mercedes-benz/sechub/files/9202484/result.zip)

Unfortunately I'll be away from my computer till next week. And I'll able to examine this problem and provide more deatiled information only next week.

**Comment by Jeeppler** [comment]
@zigfridus I am unable to add you as a reviewer. I think, I can only add reviewers which are part of the Mercedes-Benz organization.

I created already an issue for the exception you got: #1501. However, I am not 100% sure, what the root cause is.

File: sechub-pds-solutions/pmd/README.adoc [hunk 1]
```diff
@@ -0,0 +1,394 @@
+// SPDX-License-Identifier: MIT
+
+:toc:
+:numbered:
+
+= PMD + PDS
+
+https://pmd.github.io/[PMD] is a free and open source static application security testing tool. PMD is a command line interface (CLI) tool. By combining PMD with the SecHub https://mercedes-benz.github.io/sechub/latest/sechub-product-delegation-server.html[Product Delegation Server (PDS)] in a container, PMD can be used with SecHub. Another scenario is to use PMD+PDS standalone.
+
+Furthermore, the combination of PMD and PDS makes it possible to run both inside a Kubernetes cluser or virtual machines.
```
Comment (haerter-tss) [inline sechub-pds-solutions/pmd/README.adoc:10]: *cluster

**Review by haerter-tss — APPROVED** [review]
Looks good.

**Pull request merged**